src/components/Result/ResultHeader.vue

```diff
@@ -2,22 +2,25 @@
 	<div class="skewed-top-wrapper bg-primary-gradient text-white">
 		<Navigation />
 		<div
-			class="container flex justify-between items-center mx-auto pt-16 pb-32"
+			class="md:container flex flex-col md:flex-row justify-between items-center mx-auto pt-2 md:pt-16 pb-10 md:pb-32"
 		>
-			<div :class="{ 'w-2/3': percentage !== undefined }">
-				<p class="text-4xl pb-4">
-					{{ title }}
+			<div class="flex flex-col px-8 md:px-6 w-full" :class="{ 'md:w-2/3': percentage !== undefined }">
+				<p class="font-semibold md:font-normal text-center md:text-left text-2xl md:text-4xl break-words pb-2 md:pb-4"
+					:class="{'truncate': percentage !== undefined}"					
+				> 
+					{{ title }} 
 				</p>
-				<p class="text-xl">
+				<p class="hidden md:flex text-xl">
 					{{ description }}
 				</p>
 			</div>
-			<div v-if="percentage !== undefined" class="w-1/3">
+			
+			<div v-if="percentage !== undefined" class="w-full md:w-1/3 flex flex-col items-center px-8 md:px-0 mx-0 md:mx-6">
 				<div
-					class="ml-auto text-center rounded-lg bg-primary-500 w-2/3 py-4"
+					class="w-full md:w-2/3 py-1 md:py-4 md:ml-auto text-center text-xl rounded-lg bg-primary-500"
```

Comment:
Please, resolve the padding o medium devices, it does not look good. 
![image](https://user-images.githubusercontent.com/27869979/109661742-96b31280-7b6a-11eb-9210-f0c95f8a16bf.png)